### ISSUE TITLE
Feature request: password reset button in account settings

### ISSUE THREAD

**Issue by bk138** [sourcerer-io/sourcerer-app#589]
It would be quite handy to have the reset password functionality currently found at https://sourcerer.io/signin to be located also somewhere under https://sourcerer.io/settings. Like others (#425), I signed up using GitHub OAuth and lost the initial temp password mail. Took me quite a while to find out about :-).

**Comment by ShahrozTanveer** [comment]
Click On Forgot?
![image](https://user-images.githubusercontent.com/31120765/85735278-fb35ec00-b716-11ea-8e0a-79202d3dd41c.png)
input your Email address and click Reset Password. then you'll receive email to reset your password. 
![image](https://user-images.githubusercontent.com/31120765/85735472-1dc80500-b717-11ea-84cf-2060cbbda969.png)

**Comment by bk138** [comment]
Yeah, as written in my first comment, I know about this process. It's just hard to find. Currently, for logged-in users, it looks like this: 
![grafik](https://user-images.githubusercontent.com/80450/85917486-c59a1b80-b85a-11ea-88fe-150d0d288b3a.png)
